lib/data/data_sources/remote_data_source/sendbird_chat_remote_data_source.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+
+import 'package:sendbird_sdk/sendbird_sdk.dart';
+import 'package:sendbirdtutorial/locator/locator.dart';
+import 'sendbird_channels_remote_data_source.dart';
+
+class SendbirdChatRemoteDataSource with ChannelEventHandler {
```

Comment:
can we split the datasources and the repositories by concepts/entities/models, I would have ChatDataSource UsersDataSource and AuthenticationDataSource with the same logic for repositories